collector/src/descriptor/kind/mod.rs

```diff
@@ -16,6 +17,8 @@ use serde::{Deserialize, Serialize};
 
 use crate::error::{Error, ErrorKind};
 
+use self::bridge_network_status::BridgeNetworkStatus;
```

Comment:
can you make that a `pub use`, remove the self, and group if with other documents above?